### ISSUE TITLE
11.03 DetailActivity and Binding

### ISSUE THREAD

**Issue by MDJ2014** [udacity/ud851-Sunshine#72]
There seems to be a file missing.. I'm getting 'Cannont resolve symbol ActivityDetailBinding'
Then I try to import import com.example.android.sunshine.databinding.ActivityDetailBinding; and it is unrecognized.

**Comment by agantz** [comment]
same here you need to modify first the primary_weather_info inside the detailactivity layout

**Comment by baghaii** [comment]
I agree. The exercise seems all out of order. The layout tasks should be completed first. Someone should renumber the TODOs.

**Comment by shakesgar** [comment]
ActivityDetailBinding gets generated for you. Make sure to rebuild project after this lesson if ActivityDetailBinding is not getting generated
